data-gen/src/main/java/io/smartcat/data/loader/RandomBuilder.java

```diff
@@ -59,12 +66,12 @@ public RandomBuilder(Class<T> objectType, Randomizer random) {
     }
 
     public RandomBuilder<T> randomFromRange(String fieldName, Long lower, Long upper) {
-        fieldRules.put(fieldName, RangeRule.withRanges(lower, upper).withRandom(random));
+        fieldRules.put(fieldName, RangeRuleLong.withRanges(lower, upper).withRandom(random));
         return this;
     }
 
     public RandomBuilder<T> exclusiveRandomFromRange(String fieldName, Long lower, Long upper) {
```

Comment:
I guess these 2 method definitions should be different since boundary params are Long and return type is generic?